test/json_client_test.rb

```diff
@@ -33,6 +33,14 @@ def test_long_get_requests_timeout
     end
   end
 
+  def test_long_connections_timeout
+    url = "http://www.example.com/timeout.json"
+    stub_request(:get, url).to_raise(Net::OpenTimeout)
```

Comment:
I don't think `Net::OpenTimeout` exists in ruby 1.9.3 (hence the test failures).